theano/sandbox/gpuarray/opt.py

```diff
@@ -806,6 +806,9 @@ def local_error_convop(node, context_name):
             AbstractConv2d_gradWeights,
             AbstractConv2d_gradInputs])
 def local_lift_abstractconv2d(node, context_name):
+    if isinstance(node.outputs[0].type, GpuArrayType):
+        # Don't handle this node here.
```

Comment:
As this need a test, can you change this comment to tell:

# the op is already on the GPU